src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,12 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ name, email, body }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{name}</strong>
+      {' by '}
```

Comment:
it's better to use `span`
```suggestion
      <span> by </span>
```